openassessment/templates/openassessmentblock/staff_area/oa_staff_override_assessment.html

```diff
@@ -39,7 +39,7 @@ <h3 class="message__title">{% trans "We could not submit your assessment" %}</h3
                 <ul class="list list--actions">
                     <li class="list--actions__item">
                         <button type="submit" class="action action--submit is--disabled">
-                            <span class="copy">{% trans "Submit your assessment" %}</span>
+                            <span class="copy">{% trans "Submit Assessment" %}</span>
```

Comment:
This update makes this text and https://github.com/edx/edx-ora2/blob/efischer/new_bok_choy/openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners_assessment.html#L42 consistent.